src/web/components/SharingPermission/ParticipantSearchBar.tsx

```diff
@@ -43,19 +48,19 @@ export function ParticipantSearchBar({
   );
 
   return (
-    <div className={clsx('search-bar', { clicked: dropdownOpen })}>
+    <div className={clsx('search-bar', { clicked: open })}>
       <div className='search-bar-input-container'>
         <input
           type='text'
           className='search-bar-input'
-          onClick={() => setDropdownOpen(true)}
+          onClick={() => onToggle(true)}
           onChange={handleFilterTextChange}
-          placeholder='Search and Add Participants'
+          placeholder='Search Participants'
```

Comment:
Is it possible to also display this placeholder when the the search is `open`? That's what the design shows: 
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/d2cc1143-c881-45a0-a2d1-b22ec715e6ee)

Reply:
Yes, we used to get feedback about should not show it, so I set it to transparent. I removed css for that now